app/jobs/zipmaker_job.rb

```diff
@@ -9,6 +10,26 @@ class ZipmakerJob < ApplicationJob
   # @param [String] druid
   # @param [Integer] version
   def perform(druid, version)
+    binary_path = Moab::StorageServices.object_version_path(druid, version)
+    zip_path = DruidVersionZip.new(druid, version).file_path
+    unless File.exist?(zip_path)
+      ZipmakerJob.zip_binary(zip_path, binary_path) if binary_path
+    end
     PlexerJob.perform_later(druid, version)
   end
+
+  # @param [String] path to druid version zip
```

Comment:
super nit-picky;  looking at these comments with fresh eyes, they are a bit confusing.  how about something more like
- "path to zip file we are making" 
and
- "path to druid's Moab version directory" 
???

Here and below ...